scripts/__fixtures__/correct.tsx

```
@@ -0,0 +1,33 @@
+
+  React.useEffect(() => {
+    toast.error(t('errors.somethingWrong'));
+    toast.success(t('success.operationCompleted'));
+    toast.warning(t('warnings.checkInput'));
+    toast.info(t('info.updateAvailable'));
+  }, [t]);
+
+  return (
+    <div>
+      <h1>{t('dashboard.welcome')}</h1>
+      <button>{t('common.clickMe')}</button>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)